Availability for records in temporary locations

hectorcorrea opened the pull request:
Use 'Check record' as the label for records in temp locations since otherwise they stay with the 'Loading...' label. Notice that we cannot determine the availability for these records because Alma does not return it to us on the normal bib level API call. In the show page we make a second (and slow-ish) API call to fetch the complete information and hence the hint to "Check Record" to allow the user to view all the information if they are indeed interested in this record.

Fixes #2524

![temp_location_av](https://user-images.githubusercontent.com/568286/122985548-53282280-d36c-11eb-9c82-81f1af0ff869.png)

hectorcorrea commented on app/javascript/orangelight/availability.es6:
```diff
@@ -123,15 +123,25 @@ export default class AvailabilityUpdater {
       const availability_element = $(`*[data-availability-record='true'][data-record-id='${record_id}'][data-holding-id='${holding_id}'] .availability-icon`);
 
-      if (availability_info['temp_loc']) {
+      if (availability_info['temp_location']) {
```
This is unrelated to this PR and the `if` block is probably not even needed anymore, but at least now it uses the correct property name (`temp_location`)

hackartisan reviewed (changes requested):
Thank you for including the screen shot. When I looked at it, my first thought was "check record for what?" since it's the first result and I didn't have the context of "Available" or "Not Available" to slot the new value into. I wonder if it should say "Check record for Availability"?

Also you know I am going to ask! How feasible would it be to add a test for this change?

hectorcorrea commented:
@hackartisan I changed the label (see picture below) and added a test for it.

![temp_location_av](https://user-images.githubusercontent.com/568286/123138180-beccc700-d422-11eb-8107-4d9caa329535.png)

hackartisan reviewed (approved):
Great, thanks!
